baseplate/sidecars/live_data_watcher.py

```diff
@@ -85,7 +85,13 @@ def get_data_to_write(json_data: dict, data: Optional[bytes]) -> Optional[bytes]
         # }
         # If the load type is 'S3', this format is an indication that we support
         # downloading the contents of encrypted files uploaded to S3.
-        if json_data.get("live_data_watcher_load_type") == "S3":
+        is_s3 = False
+        try:
+            is_s3 = json_data["live_data_watcher_load_type"] == "S3"
+        except (KeyError, TypeError):
+            is_s3 = False
+
+        if is_s3:
             if not S3_FETCHER_ENABLED:
```

Comment:
Thinking about this now and based on the discussion we had, I think we should consider requiring boto3 as a dependency so that data writers can move between zk and s3 at will without breaking folks that don't have it installed.